labellab-client/src/actions/auth.js

```diff
@@ -28,9 +38,13 @@ export const login = (username, password, callback) => {
     dispatch(request())
     FetchApi('POST', '/api/v1/auth/login', data)
       .then(res => {
-        if (res.data && res.data.token) {
-          setToken(TOKEN_TYPE, res.data.token)
-          dispatch(success(res.data.token))
+        if (res.data && res.data.access_token && res.data.refresh_token) {
+          const { access_token, refresh_token, body } = res.data
+          // Set token to Auth header
+          setAuthToken(access_token)
+          //Save to localstorage
+          saveAllTokens({ access_token, refresh_token, body })
```

Comment:
setAuthToken() saveAllTokens() removeAllTokens() functions are not defined. did you run your code once? even this doesn't build

Reply:
![image](https://user-images.githubusercontent.com/27926576/87218371-9b268300-c36f-11ea-9701-50e19b34b2bf.png)

Reply:
Sorry I usually work on a local branch for testing all the frontend first with the latest changes in the backend and make changes in the then push it to a different branch and in that process some of the changes I may make mistake while resolving conflicts as I cant test on the oauth branch in itself as it does not have the other changes of the app. I will try to avoid such mistakes in the future.

Reply:
I would recommend rebasing the revamped frontend PR in order to avoid some errors or we could merge that also.